maas-schemas/schemas/core/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Hmmm maybe use another name, or it'll mix up with Node's package.json 😆

Reply:
So far I didn't see any issue... any suggestions ? 
`package-option` ?

Reply:
that's fine by me 👍